### PR TITLE
Add delete buttons to each comment added to the webpage.

### PR DESCRIPTION
**Pull request by neelbagora**
Comments in .java follow the format specified in the google link, the .js comments are still a work in progress.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -17,127 +17,225 @@ const AIRPLANES = "These are photos I have taken from the flight simulator, X-Pl
...
+ * validateForm ensures that the input in the comment form contains valid
+ * values.
+ *
+ * @return boolean expression verifying that input is correct.
+ */
+function validateForm() {
+	var fname = document.forms["comment-form"]["fname"].value;
+	var message = document.forms["comment-form"]["message"].value;
+	if (fname == "") {
+			alert("First Name is empty!");
```
Comment (avenkat-google) [inline portfolio/src/main/webapp/script.js:187]: nit: Keep spacing/tabbing consistent.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -17,127 +17,225 @@ const AIRPLANES = "These are photos I have taken from the flight simulator, X-Pl
...
+  form.appendChild(deleteBtn);
+  return form;
+}
+
+/**
+ * function used by the limit-selector element to specify number
+ * of elements to post. Configured url_data upon change.
+ */
+function selectFunction() {
+    var selection = document.getElementById("limit-selector").value;
```
Comment (avenkat-google) [inline portfolio/src/main/webapp/script.js:168]: nit: Keep spacing/tabbing consistent.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (neelbagora) [reply portfolio/src/main/webapp/script.js:187]: Just fixed spacing.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,119 +19,206 @@
...
+   * 
+   * <b>This method will log IP addresses making requests to the server.</b>
+   * 
+   * @param request HttpServletRequest used to make POST request.
+   * @param response HttpServletResponse used in GET requests.
+   * @throws IOException
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+		postIpAddress(getClientIpAddress(request));
```
Comment (avenkat-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: This indentation for example should be 2: https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

**Pull request merged**